updater/updater.php

```diff
@@ -113,19 +113,40 @@
 		//it means the token is expired, so ask for re-entry
 		$loader->set_template_file("updater_screen_three");
 		$updater_token_string = "Your $updater_host User Access Token Was Expired, Re-enter it now";
-		$loader->assign("HOST",xlt($updater_host_string));
+		$loader->assign("HOST",xlt($updater_token_string));
```

Comment:
Translation of this string should occur in the construction of the string on line 115.
ie: $updater_token_string = xlt("Your") .' '. $updater_host .' '. xlt("User Access Token Was Expired, Re-enter it now")

Otherwise a translation into another language will not work.

Reply:
Even that is iffy for RTL languages and some other constructs.  but be do what we can.